python/ray/experimental/state.py

```diff
@@ -110,22 +130,21 @@ def object_table(self, object_id=None):
     self._check_connected()
     if object_id is not None:
       # Return information about a single object ID.
-      return self._object_table(object_id.id())
+      return self._object_table(object_id)
     else:
       # Return the entire object table.
-      object_info_keys = self.redis_client.keys(OBJECT_INFO_PREFIX + "*")
-      object_location_keys = self.redis_client.keys(
-          OBJECT_LOCATION_PREFIX + "*")
+      object_info_keys = self._keys(OBJECT_INFO_PREFIX + "*")
+      object_location_keys = self._keys(OBJECT_LOCATION_PREFIX + "*")
       object_ids_binary = set(
           [key[len(OBJECT_INFO_PREFIX):] for key in object_info_keys] +
           [key[len(OBJECT_LOCATION_PREFIX):] for key in object_location_keys])
       results = {}
       for object_id_binary in object_ids_binary:
         results[binary_to_object_id(object_id_binary)] = self._object_table(
-            object_id_binary)
+            ray.local_scheduler.ObjectID(object_id_binary))
```

Comment:
This is what the `binary_to_object_id` function does (which we use in the previous line)